trip-ninja-docs/src/navbar/NavBar.css

```diff
@@ -1,11 +1,84 @@
 
-.navbar {
+.navbar-top {
+  background-color: var(--grey-lighter);
+  height: 90px;
+  width: 100%;
+  overflow-x: auto;
+}
+
+
+.navbar-bottom {
   background-color: var(--grey-lighter);
   border-bottom: 1px solid var(--grey-mid);
+  height: 60px;
+  overflow-x: auto;
 }
 
 .logo-text {
   color: var(--primary);
   font-weight: 800;
   font-size: 22px; 
+}
+
+.nav-link {
+  margin-right: 10px;
+  font-weight: 500;
+  display: inline-block;
+  position: relative;
+  white-space: nowrap;
+}
+
+.nav-link:hover, .selected-link {
+  color: var(--primary) !important;
```

Comment:
I think Bootstrap uses .active to set the selected link. Might be able to leverage that or at least increase the CSS specificity so we can avoid the !important tag

Reply:
added an id instead of class to increase specificity! setting to active still defaults to the black colour when active